hibernate-core/src/main/java/org/hibernate/dialect/OracleDialect.java

```diff
@@ -506,10 +507,10 @@ protected void registerDateTimeTypeMappings() {
 
 	protected void registerBinaryTypeMappings() {
 		registerColumnType( Types.BINARY, 2000, "raw($l)" );
-		registerColumnType( Types.BINARY, "long raw" );
+		registerColumnType( Types.BINARY, "blob" );
```

Comment:
Wasn't this already changed in https://hibernate.atlassian.net/browse/HHH-10345?

Reply:
Could be, but note that this is a patch to the Hibernate 6 branch, and that change was made to Hibernate 5.